frontend/src/components/ProjectListSidePanel.vue

```diff
@@ -52,6 +55,22 @@ export default {
         });
     });
 
+    const kbarActions = computed((): Action[] => {
+      const actions = outlineItemList.value.map((proj: any) =>
+        createAction({
+          id: `projects-${proj.id}`,
+          section: "Projects",
+          name: proj.name,
+          keywords: "projects",
```

Comment:
how about just "project" ?

Reply:
Yes maybe. How about also changing "bookmarks" -> "bookmark" and "databases" -> "database"?

Reply:
SGTM